internal/cli/cli.go

```diff
@@ -7,14 +7,18 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var debug bool
+
 func Execute() {
+
 	rootCmd := &cobra.Command{
 		Use:   "tilt",
 		Short: "tilt is great, yo",
 	}
 
 	addCommand(rootCmd, &upCmd{})
 	addCommand(rootCmd, &daemonCmd{})
+	rootCmd.PersistentFlags().BoolVarP(&debug, "debug", "d", false, "Run with verbose debug messages")
```

Comment:
Did we ever figure out how to call `SetDebugMode` appropriately here?

Reply:
See up.go :))

Reply:
Aha, brilliant!